### PR TITLE
Contacts OpenAPI Spec Uplift

### PR DESCRIPTION
**Pull request by aporciuncula**
https://whispir.atlassian.net/browse/WFT-190

https://developers.whispirdev.com/branches/WFT-190-contacts-openapi-uplift/b3A6Mzg5NDM5NDk-create-a-contact

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -8856,6 +8853,212 @@ components:
...
+                link:
+                  - uri: 'https://api.au.whispir.com/workspaces/9A4C5521FFC7B15B/messages/747AB7E716C1802B6476784AEB5C9BB8/messageresponses'
+                    rel: self
+                    method: GET
+                    host: api.au.whispir.com
+                    port: -1
+                lastModifiedTime: '2022-03-29T16:30:18+11:00'
+      headers:
+        Content-Type:
+          $ref: '#/components/headers/Template-Content-Type'
```
Comment (richardfontein) [inline openapi.yaml:8944]: This one should be the Contact-Content-Type

File: openapi.yaml [hunk 2]
```diff
@@ -8856,6 +8853,212 @@ components:
...
+                    devices: []
+                    link:
+                      - uri: 'https://api.au.whispir.com/workspaces/9A4C5521FFC7B15B/messages/747AB7E716C1802B6476784AEB5C9BB8/messageresponses'
+                        rel: self
+                        method: GET
+                        host: api.au.whispir.com
+                        port: -1
+      headers:
+        Content-Type:
+          $ref: '#/components/headers/Template-Content-Type'
```
Comment (richardfontein) [inline openapi.yaml:9053]: This one also needs the Contact-Content-Type

File: openapi.yaml [hunk 3]
```diff
@@ -8856,6 +8853,212 @@ components:
...
+                  - channel: sms
+                    enabled: true
+                    primary: WorkMobilePhone1
+                    secondary: WorkMobilePhone1
+                    tertiary: WorkMobilePhone1
+                type: PROJECT
+                mri: '61400400400'
+                devices: []
+                link:
+                  - uri: 'https://api.au.whispir.com/workspaces/9A4C5521FFC7B15B/messages/747AB7E716C1802B6476784AEB5C9BB8/messageresponses'
```
Comment (richardfontein) [inline openapi.yaml:8936]: This `link` example should have the correct Contact links array

File: openapi.yaml [hunk 4]
```diff
@@ -8856,6 +8853,212 @@ components:
...
+                      - channel: sms
+                        enabled: true
+                        primary: WorkMobilePhone1
+                        secondary: WorkMobilePhone1
+                        tertiary: WorkMobilePhone1
+                    type: PROJECT
+                    mri: '61400400400'
+                    devices: []
+                    link:
+                      - uri: 'https://api.au.whispir.com/workspaces/9A4C5521FFC7B15B/messages/747AB7E716C1802B6476784AEB5C9BB8/messageresponses'
```
Comment (richardfontein) [inline openapi.yaml:9046]: This `link` should also have the Contact link array items

**Review by richardfontein — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 1 above)
Comment (aporciuncula) [reply openapi.yaml:8944]: oh you're right! good catch!

**Review by richardfontein — CHANGES_REQUESTED** [review]
I think we may also need to remove the `overrides` applied to Contacts endpoints from the `.spectral.yaml` file

**Pull request merged**